src/packet_handlers/peer_subscription.rs

```diff
@@ -10,10 +10,41 @@ use super::game_state::block::BlockStateOperations;
 use super::game_state::player;
 use super::game_state::player::PlayerStateOperations;
 
-pub fn handle_peer_packet(packet: Packet, messenger: Sender<MessengerOperations>) {
-    //Whenever a peer we subscribe to sends us a packet, we just broadcast it to all our local
-    //players
-    broadcast_packet!(messenger, packet, None, false).unwrap();
+pub fn handle_peer_packet(
+    packet: Packet,
+    messenger: Sender<MessengerOperations>,
+    player_state: Sender<PlayerStateOperations>,
+) {
+    match packet.clone() {
+        Packet::SpawnPlayer(packet) => {
+            if packet.entity_id >= 1000 {
+                broadcast_packet!(messenger, Packet::SpawnPlayer(packet), None, false).unwrap();
```

Comment:
We don't want to spawn new entities for anchored players, since they're already spawned